site/_data/executables.json

```diff
@@ -0,0 +1,14 @@
+{
+  "_overrides": {
+    "windows": {
+      "ui": "UX.Wpf"
+    },
+    "linux": {
+      "ui": "UX.Gtk"
+    }
+  },
+  "_base": "OpenTabletDriver",
+  "console": "Console",
+  "daemon": "Daemon",
+  "ui": "UI"
```

Comment:
Is this for when Avalonia port gets merged?

Reply:
Yes, but the overrides take precedence for now. Removing the overrides reverts to using this.